src/styles/globals.css

```
@@ -1,6 +1,10 @@
-@import 'tailwindcss/base';
-@import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
+/*@import '../../node_modules/tailwindcss/base.css';*/
```

Stylelint (17.7.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)